src/components/CompaniesList/CompaniesCard/CompaniesCard.js

```diff
@@ -0,0 +1,47 @@
+import Button from "../../UI/Button/Button";
+import styles from "./CompaniesCard.module.scss";
+
+const CompaniesCard = ({ phone, email, piva, website, notes, name }) => {
```

Comment:
Maybe we can use the singular form here? Like `CompanyCard` since this is a card for a single item.